templates/form/bitcoin.liquid

```diff
@@ -12,7 +12,7 @@
       <br><br>
       <a class="btn btn-primary" href="{{ model.bitcoin_uri }}">{% t Pay using Bitcoin client %}</a>
     </p>
-    <p>{% t When finished, you will be redirected back to your receipt. %}</p>
```

Comment:
This piece of text is replicated through out the project - could you change them all?